src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -370,11 +370,18 @@ def load_arguments(self, _):
             c.argument('message_of_the_day', type=str,
                        validator=validate_message_of_the_day)
 
-    for scope in ['aks nodepool show', 'aks nodepool delete', 'aks nodepool scale', 'aks nodepool upgrade', 'aks nodepool update']:
+    for scope in ['aks nodepool show', 'aks nodepool scale', 'aks nodepool upgrade', 'aks nodepool update']:
         with self.argument_context(scope) as c:
             c.argument('nodepool_name', type=str, options_list=[
                        '--name', '-n'], validator=validate_nodepool_name, help='The node pool name.')
 
+    with self.argument_context('aks nodepool delete') as c:
+        c.argument('nodepool_name', type=str, options_list=[
+            '--name', '-n'], validator=validate_nodepool_name, help='The node pool name.')
```

Comment:
Since the default value of `type` is `str`, `type=str` does not need to be defined repeatedly
```suggestion
        c.argument('nodepool_name', options_list=[
            '--name', '-n'], validator=validate_nodepool_name, help='The node pool name.')
```